cookbook/security/user_checkers.rst

```diff
@@ -0,0 +1,215 @@
+
+            <config>
+                <!-- ... -->
+                <firewall name="secured_area" pattern="^/">
+                    <user-checker>app.user_checker</user-checker>
+                    <!-- ... -->
+                </firewall>
+            </config>
+        </srv:container>
+
+    .. code-block:: php
+
+        // app/config/security.php
+
+        // ...
+        $container->loadFromExtension('security', array(
+            'firewalls' => array(
+                'secured_area' => array(
+                    'pattern' => '^/',
+                    'user_checker' => 'app.user_checker',
+                    // ...
+                ),
+            ),
+        ));
+
+
+Additional Configurations
+-------------------------
+
+It's possible to have a different user checker per firewall.
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # app/config/security.yml
+
+        # ...
+        security:
+            firewalls:
+                admin:
+                    pattern: ^/admin
+                    user_checker: app.admin_user_checker
+                    # ...
+                secured_area:
+                    pattern: ^/
+                    user_checker: app.user_checker
+
+    .. code-block:: xml
+
+        <!-- app/config/security.xml -->
+        <?xml version="1.0" encoding="UTF-8"?>
+        <srv:container xmlns="http://symfony.com/schema/dic/security"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:srv="http://symfony.com/schema/dic/services"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+            <config>
+                <!-- ... -->
+                <firewall name="admin" pattern="^/admin">
+                    <user-checker>app.admin_user_checker</user-checker>
+                    <!-- ... -->
+                </firewall>
+                <firewall name="secured_area" pattern="^/">
+                    <user-checker>app.user_checker</user-checker>
+                    <!-- ... -->
+                </firewall>
+            </config>
+        </srv:container>
+
+    .. code-block:: php
+
+        // app/config/security.php
+
+        // ...
+        $container->loadFromExtension('security', array(
+            'firewalls' => array(
+                'admin' => array(
+                    'pattern' => '^/admin',
+                    'user_checkers' => 'app.admin_user_checker'
+                    // ...
+                ),
+                'secured_area' => array(
+                    'pattern' => '^/',
+                    'user_checker' => 'app.user_checker',
+                    // ...
+                ),
+            ),
+        ));
+
+.. note::
+
+    Internally the user checkers are aliased per firewall. For `secured_area` the alias
+    `security.user_checker.secured_area` would point to `app.user_checker`.
```

Comment:
use double backticks instead of single ones (single backticks is markdown syntax)